.env_sample

```diff
@@ -1,9 +1,15 @@
 export ADMIN_EMAIL=youremail@gmail.com
-export AUTH_API_KEY=2PzB7PPnpuLsbWmWtXpGyI+kfSQSQ1zUW2Atz/+8PdZuSEJzHgzGnJWV35nTKRwx/dwylauth.herokuapp.com
-export EMAIL_APP_URL=https://dwylmail.herokuapp.com
+export AUTH_URL=dwylauth.herokuapp.com
```

Comment:
Had to add the `AUTH_URL` environment variable to avoid the circular dependency on `auth_plug` noted by @tadasajon in #170